spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasLayer.scala

```diff
@@ -51,23 +67,94 @@ object KerasLayerSerializer extends ContainerSerializable with TKerasSerializerH
   }
 }
 
+private[bigdl] class KerasIdentityActivationWrapper[T: ClassTag]
+(val torchLayer: AbstractModule[Activity, Activity, T],
+    val activation: KerasLayer[Tensor[T], Tensor[T], T])(implicit ev: TensorNumeric[T])
+  extends KerasLayer[Activity, Activity, T](null) {
+  require(!torchLayer.isKerasStyle(),
+    s"We should get torch style layer, but got: $torchLayer")
+
+  override def computeOutputShape(inputShape: Shape): Shape = {
+    torchLayer.computeOutputShape(inputShape)
+  }
+
+  override def doBuild(inputShape: Shape): AbstractModule[Activity, Activity, T] = {
+    val torchActivation = activation.doBuild(inputShape)
+    require(!torchActivation.isKerasStyle(),
+      s"We should get torch style activation, but got: $torchActivation")
+    val seq = TSequential[T]()
+    seq.add(torchLayer)
+    seq.add(torchActivation)
+    seq.setName(torchLayer.getName())
+    seq
+  }
+}
+
+/**
+ * Wrap a torch style layer to keras style layer.
+ * This layer can be built multiple times.
+ * We are supposing the inputshape and the outputshape keep the same in this layer.
+ * @param layer a torch style layer
+ * @return a keras compatible layer
+ */
+class KerasIdentityWrapper[T: ClassTag]
+(val layer: AbstractModule[Activity, Activity, T])(implicit ev: TensorNumeric[T])
+  extends KerasLayer[Activity, Activity, T](null) {
+  if (layer.isKerasStyle()) {
+    throw new RuntimeException(s"We only accept torch layer here, but got: $layer")
+  }
+  override def computeOutputShape(inputShape: Shape): Shape = {
+    inputShape
+  }
+  override def doBuild(inputShape: Shape): AbstractModule[Activity, Activity, T] = layer
+
+  override def allowRebuilt(): Boolean = true
```

Comment:
Why should we allow rebuilt here? No reason to do that.

Reply:
`doBuild` just simply return parameter `layer`. In other words, there's no overhead to invoke it multiple times, I think it's safe to allow this.